tests/Controller/ProxyControllerTest.php

```diff
@@ -41,39 +47,46 @@ protected function setUp() {
 		parent::setUp();
 
 		$this->appName = 'mail';
-		$this->request = $this->getMockBuilder('\OCP\IRequest')
-			->disableOriginalConstructor()
-			->getMock();
-		$this->urlGenerator = $this->getMockBuilder('\OCP\IURLGenerator')
-			->disableOriginalConstructor()
-			->getMock();
-		$this->session = $this->getMockBuilder('\OCP\ISession')
-			->disableOriginalConstructor()
-			->getMock();
-		$this->clientService = $this->getMockBuilder('\OCP\Http\Client\IClientService')->getMock();
+		$this->request = $this->createMock([
+			IRequest::class,
+			ArrayAccess::class,
```

Comment:
This is an API issue in the server, will report this and/or prepare a PR. cc @rullzer as discussed.